satpy/readers/avhrr_l1b_gaclac.py

```diff
@@ -98,15 +98,22 @@ def __init__(self, filename, filename_info, filetype_info,  # noqa: D417
         if self._end_time < self._start_time:
             self._end_time += timedelta(days=1)
         self.platform_id = filename_info["platform_id"]
-        if self.platform_id in ["NK", "NL", "NM", "NN", "NP", "M1", "M2",
-                                "M3"]:
+
+        if len(self.platform_id) == 3:
+            self.reader_kwargs["eosip_header"] = True
+
+        if self.platform_id in ["NK", "NL", "NM", "NN", "NP",
+                                "N15", "N16", "N17", "N18", "N19",
+                                "M1", "M2", "M3",
+                                "MOB", "MOA", "MOC"]:
```

Comment:
This could be moved to a new method `_is_klm`

Reply:
good idea... calling it `_is_avhrr3`